# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/public/friend/UsageStatistics.h
# Commit: 7d099d4bac74
# Full Hash: 7d099d4bac7446d5bd798f813a2e78b1aefb1816
# Author: Denis Palmeiro <dpalmeiro@mozilla.com>
# Date: 2025-01-15 21:57:20
# Regressor Bug: 1932686
# File Overlap Count: 4
# Description:
#   Bug 1932686: Migrate enumeration legacy GC metrics to glean using a labeled counter. r=jonco,chutten
#   
#   Depends on D231645
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D231646
# ==============================================================================

diff -r 49990191195a -r 7d099d4bac74 js/public/friend/UsageStatistics.h
--- a/js/public/friend/UsageStatistics.h	Wed Jan 15 15:30:16 2025 +0000
+++ b/js/public/friend/UsageStatistics.h	Wed Jan 15 15:30:16 2025 +0000
@@ -75,7 +75,10 @@
  * However, unlike the legacy list, each glean metric must be manually added
  * to the switch statement in AccumulateTelemetryCallback().
  */
-#define FOR_EACH_JS_GLEAN_METRIC(_) _(ION_COMPILE_TIME, TimeDuration_US)
+#define FOR_EACH_JS_GLEAN_METRIC(_)    \
+  _(ION_COMPILE_TIME, TimeDuration_US) \
+  _(GC_GLEAN_SLOW_PHASE, Enumeration)  \
+  _(GC_GLEAN_SLOW_TASK, Enumeration)
 
 #define FOR_EACH_JS_METRIC(_)  \
   FOR_EACH_JS_LEGACY_METRIC(_) \